### ISSUE TITLE
Unable to log into game while able to log into website

### ISSUE THREAD

**Issue by SEKREHMJEHTH** [beyond-all-reason/Beyond-All-Reason#2908]
I can log into the website, so I know that my username and password are correct. When I try to log in, I am told I need to confirm with a verification code sent to my email, but the code is never sent. Only the first code, used for the initial setup is sent, and when I try to use that code, I am told that it is incorrect. From the increasing number of "a code has been sent to email" messages on the menu, I hypothesize that there is some bug in the sending of the emails.

**Comment by Beherith** [comment]
Username/email?

**Comment by SEKREHMJEHTH** [comment]
Username: ThermoNuclearReptile
Email: mdvork8o@gmail.com

**Comment by SEKREHMJEHTH** [comment]
Problem fixed, thank you!

**Comment by AdamLinke** [comment]
@Beherith 
Hi I am facing the same issue. Do you know how can I fix this?
Username: MeszekLozdzer
Email: gaamaal@wp.pl

**Comment by Bitcoinrocket** [comment]
Hi I am facing the same issue. Do you know how can I fix this?

Username : vesa
email : vesadn@pm.me

**Comment by kissedbysnow** [comment]
@Beherith 

Username: 961800675
Email: 961800675@qq.com

Username: kisnow
Email: atlantisdark@foxmail.com

**Comment by jeux21109** [comment]
@Beherith 

Username: jeux21109

Gmail: jeulian.axl.attard@gmail.com

**Comment by micos7** [comment]
@Beherith 

Username: micos
Gmail: micos7@gmail.com

**Comment by TCSwat101** [comment]
same issue here 

username: TCSwat
Email: Drewdymond1@gmail.com

**Comment by TristanAdams03** [comment]
Same issue here:

username: LiveHotDawg

email: joeshamrock2@gmail.com

**Comment by p2004a** [comment]
Folks: sharing your email adresses here in public will just make you get more spam.

To get help with account issues at the moment you have to join BAR Discord and open a ticket in `#open-ticket` section.

**Comment by key0013** [comment]
Title: Unable to login and issues with alternative account

Description:
I am experiencing issues with logging into my main account as well as managing an alternative account.

Main Account Issue: When attempting to log in using the username dim, I receive an "Invalid password" error even though I am certain that I have entered the correct password.

Password Reset Issue: When trying to reset my password, I get a message saying "No user by that email."

Alternative Account Issue: I was able to register a new alternative account with the username rlaus. Initially, it worked fine. However, after some time, I received a notification stating that an alternative account is forbidden and I should log in under my main account.

Discord support is unavailable, hence reporting here.
Main Username: dim
Alternative Username: rlaus
